views/admin/user/index.html

```diff
@@ -10,15 +10,15 @@
         {% for user in users %}
             <tr>
                 <td class="col-userName">
-                    <i class="icon-user"></i> <strong>{{ user.userName }}</strong><br />
-                    {{ user.displayName }} <a href="mailto:{{ user.email }}">&lt;{{ user.email }}&gt;</a>
+                    <i class="icon-user"></i> <strong>{{ user.userInfo.userName }}</strong><br />
```

Comment:
I think this structure is weird. Is there something else we can do? Maybe add `calendarCount` and `addressBookCount` as separate arrays, or add each of those properties to the users object?